frontend/packages/rhoas-plugin/locales/en/rhoas-plugin.json

```diff
@@ -1,44 +1,44 @@
+  "Discover managed services to simplify deployments and reduce operational overhead & complexities": "Discover managed services to simplify deployments and reduce operational overhead & complexities",
+  "Browse managed services to connect applications and microservices to support services to create a full solution.": "Browse managed services to connect applications and microservices to support services to create a full solution.",
   "Unlock with token": "Unlock with token",
   "Unlocked": "Unlocked",
-  "RHOAS can include Streams for Kafka, Service Registry": "RHOAS can include Kafka, Service Registry and other Red Hat Services",
-  "KafkaCardDescription": "OpenShift Streams for Apache Kafka",
-  "Access Red Hat Cloud Services with API Token": "Access Red Hat Cloud Services with API Token",
-  "ManagedServices-card-description": "Red Hat OpenShift Application Services provide a streamlined developer experience for building, deploying, and scaling cloud-native applications in open hybrid-cloud environments. The combination of enterprise-grade Kubernetes, cloud-native approach toapplication delivery, and managed operations allows teams to focus on core competencies, accelerate time to value and reduce operational cost.",
-  "KafkaStreamsService": "Streams for Apache Kafka",
-  "KafkaStreamsServiceDescription": "Streams for Apache Kafka",
+  "Red Hat OpenShift Application Services include Red Hat OpenShift API Management and Red Hat OpenShift Streams for Apache Kafka": "Red Hat OpenShift Application Services include Red Hat OpenShift API Management and Red Hat OpenShift Streams for Apache Kafka",
+  "Red Hat OpenShift Application Services include:\n- Red Hat OpenShift API Management\n- Red Hat OpenShift Streams for Apache Kafka": "Red Hat OpenShift Application Services include:\n- Red Hat OpenShift API Management\n- Red Hat OpenShift Streams for Apache Kafka",
+  "Red Hat OpenShift Application Services include:": "Red Hat OpenShift Application Services include:",
+  "Red Hat OpenShift API Management": "Red Hat OpenShift API Management",
+  "Cloud Services Card Description": "Red Hat OpenShift Application Services provide a streamlined developer experience for building, deploying, and scaling cloud-native applications in open hybrid-cloud environments. The combination of enterprise-grade Kubernetes, cloud-native approach to application delivery, and managed operations allows teams to focus on core competencies, accelerate time to value and reduce operational cost.",
+  "Unlock with API token": "Unlock with API token",
   "Connect": "Connect",
-  "Red Hat Cloud Services": "Red Hat Cloud Services",
-  "Problem with creating secret": "Problem with creating secret: {{error}} Namespace: {{namespace}}",
```

Comment:
Those were valid cases that were removed. We use error in proper context here etc. 
I have shown the context on the call so hopefully it will be clear. We need this kind of errors that give users information. 

-1 for removing existing error handling

Reply:
Added back `  "Problem with creating secret": "Problem with creating secret: {{error}} Namespace: {{namespace}}",`